CDBTo3DTiles/src/Gltf.h

```diff
@@ -9,10 +12,15 @@
 
 namespace CDBTo3DTiles {
 
-tinygltf::Model createGltf(const Mesh &mesh, const Material *material, const Texture *texture);
+tinygltf::Model createGltf(const Mesh &mesh, const Material *material, const Texture *texture, const Texture *featureIdTexture);
```

Comment:
Why isn't this an optional param? Making it required means creating calls with dummy nullptrs.